packages/runtime/datastore/src/channelDeltaConnection.ts

```diff
@@ -36,7 +37,18 @@ export class ChannelDeltaConnection implements IDeltaConnection {
     }
 
     public process(message: ISequencedDocumentMessage, local: boolean, localOpMetadata: unknown) {
-        this.handler.process(message, local, localOpMetadata);
+        try {
+            this.handler.process(message, local, localOpMetadata);
+        } catch (error) {
+            throw CreateCorruptionError(error, {
+                messageClientId: message.clientId,
+                sequenceNumber: message.sequenceNumber,
+                clientSequenceNumber: message.clientSequenceNumber,
+                referenceSequenceNumber: message.referenceSequenceNumber,
+                minimumSequenceNumber: message.minimumSequenceNumber,
+                messageTimestamp: message.timestamp,
```

Comment:
@anthony-murphy , in the latest, I'm adding some op sequence info -- I hope that `sequenceNumber` in particular will enable us to keep us from retriggering alerts in response to the same document's corruption for the same reason (as users continue to try to load it).  I think a document-id would be useful too, but I don't see where it would come from.  Also, I have `clientId` in the [dataStoreRuntime.ts](https://github.com/microsoft/FluidFramework/pull/5144/files#diff-5b145dfa0151b5be1f17d42f73c3aa248d3600b934665f3d22311345d32df4e4R589), but not here....  Will the document and client Ids be introduced higher in the stack?